pkg/acsengine/const.go

```diff
@@ -109,7 +109,7 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"addonresizer": "addon-resizer:2.0",
 		"heapster":     "heapster:v1.4.0",
 		"dns":          "k8s-dns-kube-dns-amd64:1.14.4",
-		"addonmanager": "kube-addon-manager-amd64:v6.1.2",
+		"addonmanager": "kube-addon-manager-amd64:v6.4-beta.2",
```

Comment:
It's the version that currently supports using kubectl 1.6 which is required for creation of RBAC resource types using addon-manager . See this for the changelog --
 https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/addon-manager/CHANGELOG.md#version-64-beta2--mon-june-12-2017-jeff-grafton-jgraftongooglecom. We used the 6.4 tag but which I thought would cover this beta since 6.4 hasn't actually been releases and the timestamp on 6.4 is much older than the 6.4-beta.2 -- https://gcr.io/v2/google_containers/kube-addon-manager/tags/list